### PR TITLE
release 3.0.1 - updates and examples to add cacerts

### PR DESCRIPTION
**Pull request by alknopfler**
- Add docs to include the custom cacerts
- add an example of secret to be added to the kubernetes manifest folder
- Added the changes related to 3.0.1

### CODE REVIEW

File: telco-examples/mgmt-cluster/multi-node/README.md [hunk 1]
```diff
@@ -59,6 +59,43 @@ You need to modify the following folder:
...
+global:
+  additionalTrustedCAs: true
+```
+
+2. If you are deploying a mgmt-cluster from scratch using EIB, then add the secret to the manifests folder `kubernetes/manifests/metal3-cacert-secret.yaml` to automate the creation of the secret in the management cluster:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
```
Comment (Kristian-ZH) [inline telco-examples/mgmt-cluster/multi-node/README.md:80]: The file is not formatted correctly

File: telco-examples/mgmt-cluster/multi-node/README.md [hunk 2]
```diff
@@ -59,6 +59,43 @@ You need to modify the following folder:
...
+name: metal3-system
+---
+apiVersion: v1
+kind: Secret
```
Comment (Kristian-ZH) [inline telco-examples/mgmt-cluster/multi-node/README.md:84]: The file is not formatted correctly

**Review by Kristian-ZH — CHANGES_REQUESTED** [review]

File: telco-examples/mgmt-cluster/single-node/README.md [hunk 3]
```diff
@@ -40,6 +40,45 @@ You need to modify the following folder:
 
 - `base-images` - To include inside the `SLE-Micro.x86_64-5.5.0-Default-SelfInstall-GM2.install.iso` image downloaded from the SUSE Customer Center.
 
+## Optional modifications
+
+### Add certificates to use HTTPS server to provide images using TLS
+
+This is an optional step to add certificates to the management cluster to provide images using HTTPS (Helm Chart metal3 Version >= 0.7.1)
```
Comment (Kristian-ZH) [inline telco-examples/mgmt-cluster/single-node/README.md:47]:
```suggestion
This is an optional step to add certificates to the management cluster to provide images using HTTPS Server(Helm Chart metal3 Version >= 0.7.1)
```

File: telco-examples/mgmt-cluster/multi-node/README.md [hunk 4]
```diff
@@ -59,6 +59,43 @@ You need to modify the following folder:
 
 - `base-images` - To include inside the `SLE-Micro.x86_64-5.5.0-Default-SelfInstall-GM2.install.iso` image downloaded from the SUSE Customer Center.
 
+## Optional modifications
+
+### Add certificates to use HTTPS server to provide images using TLS
+
+This is an optional step to add certificates to the management cluster to provide images using HTTPS (Helm Chart metal3 Version >= 0.7.1)
```
[the remark this answers is not in the document]
Comment (alknopfler) [reply telco-examples/mgmt-cluster/multi-node/README.md:66]: changed! thanks

**Comment by alknopfler** [comment]
changes done!

**Review by Kristian-ZH — APPROVED** [review]

**Review by hardys — APPROVED** [review]

**Pull request merged**